app/src/main/java/com/guerinet/mymartlet/util/dbflow/databases/CoursesDB.java

```diff
@@ -46,15 +47,12 @@
      * @param callback Optional callback called when the transaction is finished
      */
     public static void setCourses(Term term, List<Course> courses,
-            @Nullable DBUtils.Callback callback) {
-        if (courses == null) {
-            courses = new ArrayList<>();
-        }
+                                  @Nullable DBUtils.Callback callback) {
+
+        if (courses == null) courses = new ArrayList<>();
```

Comment:
Continuation of formatting preferences: 
- Params alignment, see first comment above 
- Null check: I always put brackets and multi-line around if/while/for... statements for clarity and consistency across the project. It also makes adding more code to the initial if statement if necessary much easier.